project/src/main/webapp/script.js

```diff
@@ -183,10 +183,13 @@ function getTrendBubbles(content, colors) {
   const bubbleSecondRow = document.getElementById("frequency-row-2");
   const bubbleSizes = getTrendBubbleSize(content);
   const sentimentScore = getTrendBubbleScore(content);
+  console.log(sentimentScore);
```

Comment:
I see a bunch of log statements in this file...any reason why they're not getting caught by the linters?

Reply:
hmm not sure. When I added prettier, it disabled some of the linter rules that conflict with prettier's, so it may have been turned off?